src/interfaces/playground.d.ts

```diff
@@ -6,6 +6,7 @@ interface ITemplate {
     ///////////////////////////////////////////////////////////////////////////////////////////////////
     id?: string;
     gist?: string;
+    owned?: boolean;
```

Comment:
We should follow standard naming conventions throughout the project. I think this should be named isOwned